### ISSUE TITLE
Error when trying to upload screenshot images

### ISSUE THREAD

**Issue by joosthaneveer** [Microsoft/google-play-vsts-extension#29]
When trying to update the metadata in the store with screenshot images it fails to upload the screenshots because the path used in the task is incorrect.

I included some debug information from my build process to clarify the bug.
For screenshot files only the filename is included in the path variable of the upload request. All other images files have the complete path to the file specified.

```
2016-09-15T07:25:00.7274119Z ##[debug]Starting upload of images with language code en-GB from C:\a\1\s\src\ProjectName\ProjectName.Droid\StoreMetadata\en-GB
2016-09-15T07:25:00.7284103Z ##[debug]Attempting to get images of type featureGraphic
2016-09-15T07:25:00.7284103Z Found image for type featureGraphic at C:\a\1\s\src\ProjectName\ProjectName.Droid\StoreMetadata\en-GB\images\featureGraphic.png
2016-09-15T07:25:00.7284103Z ##[debug]Attempting to get images of type icon
2016-09-15T07:25:00.7284103Z Found image for type icon at C:\a\1\s\src\ProjectName\ProjectName.Droid\StoreMetadata\en-GB\images\icon.png
2016-09-15T07:25:00.7284103Z ##[debug]Attempting to get images of type promoGraphic
2016-09-15T07:25:00.7284103Z ##[debug]File C:\a\1\s\src\ProjectName\ProjectName.Droid\StoreMetadata\en-GB\images\promoGraphic.png doesn't exist. Skipping...
2016-09-15T07:25:00.7284103Z ##[debug]File C:\a\1\s\src\ProjectName\ProjectName.Droid\StoreMetadata\en-GB\images\promoGraphic.jpg doesn't exist. Skipping...
2016-09-15T07:25:00.7294088Z ##[debug]File C:\a\1\s\src\ProjectName\ProjectName.Droid\StoreMetadata\en-GB\images\promoGraphic.jpeg doesn't exist. Skipping...
2016-09-15T07:25:00.7294088Z Image for promoGraphic was not found. Skipping...
2016-09-15T07:25:00.7294088Z ##[debug]Attempting to get images of type tvBanner
2016-09-15T07:25:00.7294088Z ##[debug]File C:\a\1\s\src\ProjectName\ProjectName.Droid\StoreMetadata\en-GB\images\tvBanner.png doesn't exist. Skipping...
2016-09-15T07:25:00.7294088Z ##[debug]File C:\a\1\s\src\ProjectName\ProjectName.Droid\StoreMetadata\en-GB\images\tvBanner.jpg doesn't exist. Skipping...
2016-09-15T07:25:00.7294088Z ##[debug]File C:\a\1\s\src\ProjectName\ProjectName.Droid\StoreMetadata\en-GB\images\tvBanner.jpeg doesn't exist. Skipping...
2016-09-15T07:25:00.7294088Z Image for tvBanner was not found. Skipping...
2016-09-15T07:25:00.7294088Z ##[debug]Attempting to get images of type phoneScreenshots
2016-09-15T07:25:00.7294088Z ##[debug]Found something for type phoneScreenshots
2016-09-15T07:25:00.7304111Z ##[debug]Attempting to get images of type sevenInchScreenshots
2016-09-15T07:25:00.7304111Z ##[debug]Error: ENOENT: no such file or directory, stat 'C:\a\1\s\src\ProjectName\ProjectName.Droid\StoreMetadata\en-GB\images\sevenInchScreenshots'
2016-09-15T07:25:00.7304111Z Image directory for sevenInchScreenshots was not found. Skipping...
2016-09-15T07:25:00.7304111Z ##[debug]Attempting to get images of type tenInchScreenshots
2016-09-15T07:25:00.7304111Z ##[debug]Error: ENOENT: no such file or directory, stat 'C:\a\1\s\src\ProjectName\ProjectName.Droid\StoreMetadata\en-GB\images\tenInchScreenshots'
2016-09-15T07:25:00.7314104Z Image directory for tenInchScreenshots was not found. Skipping...
2016-09-15T07:25:00.7314104Z ##[debug]Attempting to get images of type tvScreenshots
2016-09-15T07:25:00.7314104Z ##[debug]Error: ENOENT: no such file or directory, stat 'C:\a\1\s\src\ProjectName\ProjectName.Droid\StoreMetadata\en-GB\images\tvScreenshots'
2016-09-15T07:25:00.7314104Z Image directory for tvScreenshots was not found. Skipping...
2016-09-15T07:25:00.7314104Z ##[debug]Attempting to get images of type wearScreenshots
2016-09-15T07:25:00.7314104Z ##[debug]Error: ENOENT: no such file or directory, stat 'C:\a\1\s\src\ProjectName\ProjectName.Droid\StoreMetadata\en-GB\images\wearScreenshots'
2016-09-15T07:25:00.7314104Z Image directory for wearScreenshots was not found. Skipping...
2016-09-15T07:25:00.7314104Z ##[debug]Finished enumerating images: {"featureGraphic":["C:\\a\\1\\s\\src\\ProjectName\\ProjectName.Droid\\StoreMetadata\\en-GB\\images\\featureGraphic.png"],"icon":["C:\\a\\1\\s\\src\\ProjectName\\ProjectName.Droid\\StoreMetadata\\en-GB\\images\\icon.png"],"promoGraphic":[],"tvBanner":[],"phoneScreenshots":["MemberFirmList.jpg","Splash.jpg"],"sevenInchScreenshots":[],"tenInchScreenshots":[],"tvScreenshots":[],"wearScreenshots":[]}
2016-09-15T07:25:00.7314104Z ##[debug]All image uploads queued
2016-09-15T07:25:00.7314104Z ##[debug]Uploading image of type featureGraphic from C:\a\1\s\src\ProjectName\ProjectName.Droid\StoreMetadata\en-GB\images\featureGraphic.png
2016-09-15T07:25:00.7324115Z ##[debug]Making image upload request: {"language":"en-GB","imageType":"featureGraphic","uploadType":"media","media":{"body":{"_readableState":{"objectMode":false,"highWaterMark":65536,"buffer":[],"length":0,"pipes":null,"pipesCount":0,"flowing":null,"ended":false,"endEmitted":false,"reading":false,"sync":true,"needReadable":false,"emittedReadable":false,"readableListening":false,"resumeScheduled":false,"defaultEncoding":"utf8","ranOut":false,"awaitDrain":0,"readingMore":false,"decoder":null,"encoding":null},"readable":true,"domain":null,"_events":{},"_eventsCount":1,"path":"C:\\a\\1\\s\\src\\ProjectName\\ProjectName.Droid\\StoreMetadata\\en-GB\\images\\featureGraphic.png","fd":null,"flags":"r","mode":438,"autoClose":true},"mimeType":"image/png"}}
2016-09-15T07:25:03.1833761Z ##[debug]Uploading image of type icon from C:\a\1\s\src\ProjectName\ProjectName.Droid\StoreMetadata\en-GB\images\icon.png
2016-09-15T07:25:03.1843760Z ##[debug]Making image upload request: {"language":"en-GB","imageType":"icon","uploadType":"media","media":{"body":{"_readableState":{"objectMode":false,"highWaterMark":65536,"buffer":[],"length":0,"pipes":null,"pipesCount":0,"flowing":null,"ended":false,"endEmitted":false,"reading":false,"sync":true,"needReadable":false,"emittedReadable":false,"readableListening":false,"resumeScheduled":false,"defaultEncoding":"utf8","ranOut":false,"awaitDrain":0,"readingMore":false,"decoder":null,"encoding":null},"readable":true,"domain":null,"_events":{},"_eventsCount":1,"path":"C:\\a\\1\\s\\src\\ProjectName\\ProjectName.Droid\\StoreMetadata\\en-GB\\images\\icon.png","fd":null,"flags":"r","mode":438,"autoClose":true},"mimeType":"image/png"}}
2016-09-15T07:25:05.0888611Z ##[debug]Uploading image of type phoneScreenshots from MemberFirmList.jpg
2016-09-15T07:25:05.0888611Z ##[debug]Making image upload request: {"language":"en-GB","imageType":"phoneScreenshots","uploadType":"media","media":{"body":{"_readableState":{"objectMode":false,"highWaterMark":65536,"buffer":[],"length":0,"pipes":null,"pipesCount":0,"flowing":null,"ended":false,"endEmitted":false,"reading":false,"sync":true,"needReadable":false,"emittedReadable":false,"readableListening":false,"resumeScheduled":false,"defaultEncoding":"utf8","ranOut":false,"awaitDrain":0,"readingMore":false,"decoder":null,"encoding":null},"readable":true,"domain":null,"_events":{},"_eventsCount":1,"path":"MemberFirmList.jpg","fd":null,"flags":"r","mode":438,"autoClose":true},"mimeType":"image/jpeg"}}
2016-09-15T07:25:05.0918681Z events.js:154
2016-09-15T07:25:05.0918681Z       throw er; // Unhandled 'error' event
2016-09-15T07:25:05.0918681Z       ^
2016-09-15T07:25:05.0918681Z 
2016-09-15T07:25:05.0918681Z Error: ENOENT: no such file or directory, open 'C:\a\1\s\MemberFirmList.jpg'
2016-09-15T07:25:05.0918681Z     at Error (native)
2016-09-15T07:25:05.1168601Z ##[error]Exit code 1 returned from process: file name 'C:\LR\MMS\Services\Mms\TaskAgentProvisioner\Tools\agents\2.105.6\externals\node\bin\node.exe', arguments '"C:\a\_tasks\GooglePlayRelease_8cf7cac0-620b-11e5-b4cf-8565e60f4d27\1.2.3\GooglePlay.js"'.
2016-09-15T07:25:05.1188605Z ##[debug]Microsoft.VisualStudio.Services.Agent.ProcessExitCodeException: Exit code 1 returned from process: file name 'C:\LR\MMS\Services\Mms\TaskAgentProvisioner\Tools\agents\2.105.6\externals\node\bin\node.exe', arguments '"C:\a\_tasks\GooglePlayRelease_8cf7cac0-620b-11e5-b4cf-8565e60f4d27\1.2.3\GooglePlay.js"'.
```

**Comment by madhurig** [comment]
@joosthaneveer: Thanks for reporting the issue. We will track this for the next update.

**Comment by madhurig** [comment]
Fix is included in 1.3.0 release: https://marketplace.visualstudio.com/items?itemName=ms-vsclient.google-play
